examples/v2/chat2.nim

```diff
@@ -172,10 +176,24 @@ proc processInput(rfd: AsyncFD, rng: ref BrHmacDrbgContext) {.async.} =
   let listenStr = $peerInfo.addrs[0] & "/p2p/" & $peerInfo.peerId
   echo &"Listening on\n {listenStr}"
 
+  if conf.storenode != ""
```

Comment:
This is correct behavior, just like it is (or should be) in wakunode2 and https://github.com/status-im/nim-waku/blob/34a8af4cc11387078b046c37a984ea03bab90b56/examples/v2/chat2.nim#L172